hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/FederationInterceptor.java

```diff
@@ -897,9 +897,16 @@ public void setNextInterceptor(RequestInterceptor next) {
    * release all the resources that we are holding in this call.
    */
   @Override
-  public void shutdown() {
+  public void shutdown(boolean stop) {
```

Comment:
YARN-6848 plans to unify the Interceptor interface. Can we not change the shutdown parameters? Are there any other ways we can solve the issue?